spec/interactor/organizer_spec.rb

```diff
@@ -41,13 +41,15 @@ module Interactor
       before do
         allow(instance).to receive(:context) { context }
         allow(organizer).to receive(:organized) {
-          [interactor2, interactor3, interactor4]
+          [interactor2, :mapping1, interactor3, :mapping2, interactor4]
```

Comment:
Yep, it would. Just thought about having methods-as-simple-interactors approach as a second way to do the same. But probably it's better to keep things simple.